docs/demos/TrafficManagerProfile/session-migrate-to-anothe-region/README.md

```diff
@@ -0,0 +1,220 @@
+  name: ro-nginx-service-export-eastus2euap
+  namespace: team-a-nginx
+spec:
+  placement:
+    name: crp-team-a
+  resourceSelectors:
+    -  group: networking.fleet.azure.com
+       kind: ServiceExport
+       version: v1alpha1
+       name: nginx-service-eastus2euap
+  policy:
+    overrideRules:
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchExpressions:
+                  - key: fleet.azure.com/location
+                    operator: NotIn
+                    values:
+                      - eastus2euap
+        overrideType: Delete
+```
+
+Apply the configurations:
+```sh
+kubectl apply -f eastus2euap/ro-nginx-service-eastus2euap.yaml
+kubectl apply -f eastus2euap/ro-nginx-service-export-eastus2euap.yaml
+```
+
+## Step 2: Create another service to expose the existing **Nginx deployment** in the new region.
+
+> Note: test file located [here](../testfiles/uksouth/..).
+
+Apply the configurations:
+```sh
+kubectl apply -f uksouth/ro-nginx-service-uksouth.yaml
+kubectl apply -f uksouth/ro-nginx-service-export-uksouth.yaml
+kubectl apply -f uksouth/nginx-service-uksouth.yaml
+```
+
+## Step 3: Add label "env=prod" to uksouth clusters
+
+To ensure that `member-2` & `member-4` is recognized as a production environment, we need to add the label `env=prod` to it. This will allow the `clusterResourcePlacement` to propagate the resources correctly.
+
+```sh
+kubectl label cluster aks-member-2 env=prod
+kubectl label cluster aks-member-4 env=prod
+```
+
+To check the status of the `clusterResourcePlacement`, we will run the following command. It's essential to ensure the resources are applied and available on both clusters before we shift the traffic.
+
+```sh
+kubectl get crp crp-team-a -o yaml
+```
+
+---
+
+## Step 4: Add a New Traffic Manager Backend
+- Once resources are ready, add a new `trafficManagerBackend` linked to the existing `trafficManagerProfile`.
+
+> Note: test file located [here](../testfiles/nginx-backend-uksouth.yaml).
+
+```yaml
+apiVersion: networking.fleet.azure.com/v1beta1
+kind: TrafficManagerBackend
+metadata:
+  name: nginx-backend-uksouth
+  namespace: team-a-nginx
+spec:
+  profile:
+    name: "nginx-profile"
+  backend:
+    name: "nginx-service-uksouth"
+  weight: 10
+```
+
+Apply the configuration:
+```sh
+kubectl apply -f nginx-backend-uksouth.yaml
+```
+
+## Step 5: Shift Traffic Dynamically
+- Adjust backend weights in `trafficManagerBackend` to gradually shift traffic between regions.
+- Monitor traffic distribution by using `kubectl get tmb -n team-a-nginx -o yaml` to ensure a smooth transition.
+- Delete the existing `trafficManagerBackend` nginx-backend-eastus2euap for `member-1` and `member-3` so that the new connections won't be routed to the eastus2euap region.
+
+> Note: There are multiple ways to adjust the weight on the trafficManagerBackends. For example, you can fix one of weight and adjust the other one. Or you can update both.

```

Comment:
Hi Zhiying! Do we need to give people instructions on this front? Once again, not a blocker -> if we just assume that people is already familiar with this then I am good